notebooks/bedrock/Financial Multilingual Embeddings.ipynb

```diff
@@ -0,0 +1,861 @@
+      "Requirement already satisfied: tzdata>=2022.1 in /home/ec2-user/anaconda3/envs/python3/lib/python3.10/site-packages (from pandas->sagemaker->cohere-aws) (2023.3)\n",
+      "Requirement already satisfied: ppft>=1.7.6.7 in /home/ec2-user/anaconda3/envs/python3/lib/python3.10/site-packages (from pathos->sagemaker->cohere-aws) (1.7.6.7)\n",
+      "Requirement already satisfied: dill>=0.3.7 in /home/ec2-user/anaconda3/envs/python3/lib/python3.10/site-packages (from pathos->sagemaker->cohere-aws) (0.3.7)\n",
+      "Requirement already satisfied: pox>=0.3.3 in /home/ec2-user/anaconda3/envs/python3/lib/python3.10/site-packages (from pathos->sagemaker->cohere-aws) (0.3.3)\n",
+      "Requirement already satisfied: multiprocess>=0.70.15 in /home/ec2-user/anaconda3/envs/python3/lib/python3.10/site-packages (from pathos->sagemaker->cohere-aws) (0.70.15)\n",
+      "Requirement already satisfied: contextlib2>=0.5.5 in /home/ec2-user/anaconda3/envs/python3/lib/python3.10/site-packages (from schema->sagemaker->cohere-aws) (21.6.0)\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install --upgrade cohere-aws hnswlib translate"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "be54602a-944d-43c4-a611-6f8f57285618",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/ec2-user/anaconda3/envs/python3/lib/python3.10/site-packages/pandas/core/computation/expressions.py:21: UserWarning: Pandas requires version '2.8.0' or newer of 'numexpr' (version '2.7.3' currently installed).\n",
+      "  from pandas.core.computation.check import NUMEXPR_INSTALLED\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "sagemaker.config INFO - Not applying SDK defaults from location: /etc/xdg/sagemaker/config.yaml\n",
+      "sagemaker.config INFO - Not applying SDK defaults from location: /home/ec2-user/.config/sagemaker/config.yaml\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "import cohere_aws\n",
+    "import hnswlib\n",
+    "from translate import Translator\n",
+    "import warnings\n",
+    "import os\n",
+    "import re\n",
+    "import boto3\n",
+    "\n",
+    "warnings.filterwarnings('ignore')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab9e4080-542b-4aad-94f4-6cf33789c792",
+   "metadata": {},
+   "source": [
+    "### Step 2: Import Documents \n",
+    "\n",
+    "Multfin Github repo is available in https://github.com/RasmusKaer/MultiFin with a link to the data in [Dropbox](https://www.dropbox.com/sh/v0fvtn5gnf4ij6f/AABPSV4NzHIRa8GwUfQ7kA0aa?dl=0)\n",
+    "\n",
+    "Download and unzip the _CPT_Multilingual_Financial.zip_ file. Upload to your Jupyter SageMaker Notebook instance the _train.json_ file available in /MultiFinDataset_EACL/0-all-languages-lowlevel."
```

Comment:
I found the second part of this instruction confusing. Because you only mention unzipping one file, I assumed that train.json would be in that file. It would be clearer to also mention unzipping MultiFinDataset_EACL

Reply:
I improved the wording, let me know if it's better now. I also realized I was indicating to download the wrong file.